app/quests/helpers.py

```diff
@@ -188,13 +188,15 @@ def get_or_create_prize_url(quest, profile):
         btcs = BulkTransferCoupon.objects.filter(
             token=quest.kudos_reward,
             tag='quest',
+            quest_pk=quest.pk,
```

Comment:
hmmm so the only side effect of this we should be aware of is that, as of w2 jan when this is released, then all of the 100k beaten quests in the platform, will all generate new kudos reward links if the user beats them a second time.

maybe thats ok tho now that kudos are on xdai + easy/cheap to issue.  if ppl want a second reward then nbd.

Reply:
Ahh, that could be an issue, but without retroactively assigning the pk from previous quests into the db, I can't see a way to solve this in the code? 

From what I can see, it's either; allow them all to be re-rewarded, or block any that have already been rewarded from this check (ie put an or into the filter, so that we're checking for a match OR an isNull on the quest_pk)?

What would your preference be?

Reply:
> Ahh, that could be an issue, but without retroactively assigning the pk from previous quests into the db, I can't see a way to solve this in the code?

yeah, thats probably the most elegant solution.   we'd have to guess which quest it was if a user had gotten multiple bulk checkouts with the same kudos, but otherwise itd work fine

Reply:
Brilliant, I think I have a fairly concise data migration written up - I'll push it now.